dist/init/linux-systemd/caddy.service

```diff
@@ -5,18 +5,32 @@ After=network-online.target
 Wants=network-online.target systemd-networkd-wait-online.service
 
 [Service]
+; Only allow 1 restart with 10 seconds so that caddy doesn't get locked out by PAM
 Restart=on-failure
-StartLimitInterval=86400
-StartLimitBurst=5
+StartLimitInterval=10
+StartLimitBurst=1
```

Comment:
I think the reasoning for the original values was to prevent Caddy users from hitting Let's Encrypt rate limits in the case their system is misconfigured and they didn't use the `-ca` flag to set it to LE's staging endpoint. I'm not sure how valid these values are anymore... but frankly, if Caddy fails to start just once, it typically means the operator needs to fix something. There really shouldn't be a reason to try again 10 seconds later and hope it resolved itself unless, again, you're doing something funky.